### PR TITLE
[issue-1]

### PR DESCRIPTION
**Pull request by JaySunSyn**
This PR is in progress.

@plequang Please take an initial look and let me know if the way I went is ok for you.

I removed most of the api properties and need to discuss whats needed for the app.

Thanks

### CODE REVIEW

**Comment by nomego** [comment]
Why did you decide against the existing photoswipe wrapper?

**Comment by JaySunSyn** [comment]
They are actually not using photoswipe anymore. https://github.com/safetychanger/sc-photoswipe/commit/22b3464e900005e9ffb9cc92c371cbd3ab230365

File: cosmoz-image-viewer.js [hunk 1]
```diff
@@ -5,199 +5,67 @@
...
-			},
-			currentImageIndex: {
-				type: Number,
-				computed: '_computeCurrentImageIndex(currentImage, images)'
-			},
-			currentPage: {
-				type: Number,
-				computed: '_computePage(currentImageIndex)'
-			},
-			detached: {
```
Comment (nomego) [inline cosmoz-image-viewer.js]: This is not part of PhotoSwipe, it's a feature to detach the image container (also not part of photoswipe) into a separate window.

File: cosmoz-image-viewer.js [hunk 2]
```diff
@@ -5,199 +5,67 @@
...
-				observer: '_imageListChanged'
-			},
-			/**
-			 * Disable button to detach image
-			 */
-			noDetach: {
-				type: Boolean,
-				value: false
-			},
-			scrollPercent: {
```
Comment (nomego) [inline cosmoz-image-viewer.js]: Not part of PhotoSwipe, it's a setting to make the image scroll on load (for certain documents we try to scroll down past the header).

File: cosmoz-image-viewer.html [hunk 3]
```diff
@@ -5,86 +5,71 @@
...
 <link rel="import" href="../iron-resizable-behavior/iron-resizable-behavior.html" />
+
 <script src="../photoswipe/dist/photoswipe.min.js"></script>
 <script src="../photoswipe/dist/photoswipe-ui-default.min.js"></script>
 
+<link rel="import" href="photoswipe-styles.html">
+
 <dom-module id="cosmoz-image-viewer">
 	<template>
-		<style>
```
Comment (nomego) [inline cosmoz-image-viewer.html]: This template is not part of Photoswipe but how we display the image(s) inline.
Photoswipe is not activated until the "zoom" button is clicked.

**Pull request closed without merging**